app/models/mixins/miq_policy_mixin.rb

```diff
@@ -22,21 +26,14 @@ def remove_policy(policy)
     reload
   end
 
-  def get_policies(mode = nil)
-    ns = "/miq_policy"
-    cat = "assignment"
-    tag_list(:ns => ns, :cat => cat).split.collect do |t|
+  def get_policies
```

Comment:
The signature changed to remove the `mode`, but do the callers still pass a value?

Reply:
there are no callers for `get_policies(_)` - so I have 1 commit just dedicated to this

Reply:
that should read:

there are no callers into `get_policies` that pass the `mode` parameter.
So I went ahead and removed it.

This PR is about the feature but also about simplifying how we use tags in general.